packages/babel-preset-env/src/normalize-options.js

```diff
@@ -275,6 +275,11 @@ export default function normalizeOptions(opts: Options) {
     ) || corejs.proposals;
 
   return {
+    bugfixes: validateBoolOption(
+      TopLevelOptions.bugfixes,
+      opts.bugfixes,
+      false,
```

Comment:
I recall we'd said this would be enabled by default - I'm guessing this is `false` until Babel 8?

Reply:
Yes :+1:
This shouldn't technically be a breaking change, but it's safer to wait (Babel 8 will be released soonish anyway).